.changeset/thin-foxes-ring.md

```diff
@@ -0,0 +1,10 @@
+---
+'@commercetools-website/self-learning-smoke-test': patch
+'@commercetools-docs/gatsby-theme-api-docs': patch
+'@commercetools-docs/gatsby-theme-docs': patch
+'@commercetools-website/docs-smoke-test': patch
+'@commercetools-website/documentation': patch
+'@commercetools-website/site-template': patch
+---
+
+Created a dedicated smoke test site for self-learning
```

Comment:
I think we don't need a changeset here...this is just for us internally.

Reply:
You're right, but I don't want to create possible issues with other changesets, I'll leave it just in case